.github/FUNDING.yml

```diff
@@ -0,0 +1,3 @@
+# These are supported funding model platforms
+
+github: [hammy2899]
```

Comment:
Changes LGTM, but please rebase your branch to `develop` instead of `master` to avoid unnecessary commits (this file).
We only merge PR's into the develop branch.
Thank you 🙂